presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSessionProperties.java

```diff
@@ -184,6 +185,15 @@ public PrestoSparkSessionProperties(PrestoSparkConfig prestoSparkConfig)
                         true,
                         value -> MAP_SPLITTER.split(nullToEmpty((String) value)),
                         value -> value),
+                new PropertyMetadata<>(
+                        SPARK_RETRY_ON_OUT_OF_MEMORY_WITH_INCREASED_MEMORY_SETTINGS_ERROR_CODES,
+                        "Error Codes to retry with increase memory settings",
+                        VARCHAR,
+                        List.class,
+                        ImmutableList.of("EXCEEDED_LOCAL_MEMORY_LIMIT", "SPARK_EXECUTOR_OOM"),
```

Comment:
Nit: Would be good to derive these from the enum values from their respective error code provider - `StandardErrorCode.*` 
 and `SparkErrorCode.*` instead of hardcoded string values